sunpy/database/attrs.py

```diff
@@ -196,7 +196,7 @@ def _create(wlk, root, session):
             path, inverted = value
             if inverted:
                 query = query.filter(or_(
-                    DatabaseEntry.path != path, DatabaseEntry.path == None))
+                    DatabaseEntry.path != path, DatabaseEntry.path is not None))
```

Comment:
@cguennou we had this one wrong.... `== `  should be `is` instead of `is not`.  That's why all the tests are complaining :-)